volunteer.html

```diff
@@ -222,31 +225,44 @@ <h2>Advisory Board</h2>
       <div class="volunteers-container">
         <div>
           <div class="bio-with-pic">
-            <img class="bio-pic" src="assets/images/volunteers/jenh.jpg">
-            <h3>Jen Homann</h3>
+            <img class="bio-pic" src="assets/images/volunteers/melinda.jpg" alt="Melinda Deinert headshot">
+            <h3>Melinda Deinert</h3>
             <div class="badge-container">
               <div class="gwc-badge ta">[TA]</div>
               <div class="gwc-badge organizer">&lt;O&gt;</div>
             </div>
           </div>
         </div>
+
         <div>
           <div class="bio-with-pic">
-            <img class="bio-pic" src="assets/images/volunteers/hauwa.jpg">
-            <h3>Hauwa Muibi</h3>
+            <img class="bio-pic" src="assets/images/volunteers/trishag.jpg" alt="Trish Gawley headshot">
+            <h3>Trisha Gawley</h3>
             <div class="badge-container">
+              <div class="gwc-badge instructor">{I}</div>
               <div class="gwc-badge ta">[TA]</div>
               <div class="gwc-badge speaker">"S"</div>
-              <div class="gwc-badge organizer">&lt;O&gt;</div>
             </div>
           </div>
         </div>
+
         <div>
           <div class="bio-with-pic">
-            <img class="bio-pic" src="assets/images/volunteers/biancaz.jpg">
-            <h3>Bianca Zongrone</h3>
+            <img class="bio-pic" src="assets/images/volunteers/jenh.jpg">
```

Comment:
I see we weren't very good at adding alt tags before.  😓  Could you add one here, as well?

Thank you!